registry.json

```diff
@@ -1918,5 +1918,29 @@
       "rpc": "https://smartbch.fountainhead.cash/mainnet",
       "documentation": "https://github.com/smartbch/docs/blob/main/developers-guide/jsonrpc.md"
     }
+  },
+  {
+    "id": "kcc",
+    "name": "KCC",
```

Comment:
Is it the official name? not "Kucoin community Chain"?

Reply:
hello,we have change the official name with "Kucoin Community Chain"